NEWS.md

```diff
@@ -1,5 +1,20 @@
 # EpiNow2 (development version)
 
+## Model changes
+
+- The models now supports more complex patterns of aggregating reported cases by accumulating them over multiple time points, as well as mixtures of accumulation and missingness via the new `fill_missing()` function and a logical `accumulate` column that can be supplied with the data. By @sbfnk in #851 and reviewed by @seabbs and @jamesmbaazam..
```

Comment:
```suggestion
- The models now support more complex patterns of aggregating reported cases by accumulating them over multiple time points, as well as mixtures of accumulation and missingness via the new `fill_missing()` function and a logical `accumulate` column that can be supplied with the data. By @sbfnk in #851 and reviewed by @seabbs and @jamesmbaazam..
```

Reply:
as implemented it is literally accumulation and you can have multiple accumulate elements in the likelihood which would not be the same as aggregation so if we stick with that I think we need to stick with the accumulate naming

Reply:
```suggestion
- The models now support more complex patterns of aggregating reported cases by accumulating them over multiple time points, as well as mixtures of accumulation and missingness via the new `fill_missing()` function and a logical `accumulate` column that can be supplied with the data. By @sbfnk in #851 and reviewed by @seabbs and @jamesmbaazam..
```